client/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release Notes
 
+## 1.2.0 (YYYY-MM-DD)
+
+- Yseop CLI `compile` and `batch` can be used as VSCode commands.
+  - Uses the new parameter `yseopml.pathToYseopCli`
```

Comment:
Just noticed the missing period here and in the similar sentence for the path to predefined objects. It becomes quite obvious when displaying that stuff in a rendered form rather than as raw code. :wink:

Reply:
Well, the only line with a final dot is the line I added. All the other lines in this file don't have a dot.

Reply:
@mminot-yseop So, what should we do about the period thing?